files/root/grafana_tools/ldap_sync/sync.sh

```diff
@@ -17,7 +17,7 @@ do
 	if [ $1 == "--debug" ]; then
 		echo "Validating Group ${g} is still in LDAP"
 	fi
-	if [ -z "$(/usr/bin/ldapsearch -x "(cn=${g})" -b ou=groups,dc=ncsa,dc=illinois,dc=edu | grep uniqueMember)" ]; then
+        if [ -z "$(/usr/bin/ldapsearch -x "(cn=${g})" -b ou=groups,dc=ncsa,dc=illinois,dc=edu | grep uniqueMember)" ]; then
```

Comment:
nitpicking here, but i think this should have less leading spaces. 6 spaces less, i think

Reply:
I think a tab got replaced with spaces or something.

This is another one of those annoying deals where https://github.com/jdmaloney/grafana_tools is outdated. I really don't want to get nitpicky here.

Reply:
Fixed in latest commit. Reverted back to original tab instead of spaces.